### ISSUE TITLE
Wallerian degeneration

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#9734]
**Preferred term label:**
Wallerian degeneration

**Synonyms**

**Definition (free text, please give PubMed ID)**
Manifestation of distal axonal degeneration due to axonal injury at a defined site and time (PMID:24840802; 36341109).  

**Parent term (use hpo.jax.org/app)**
Abnormal nervous system morphology (HP:0012639)
or
Atrophy/Degeneration affecting the central nervous system (HP:0007367)
or
Neurodegeneration (HP:0002180) 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by pnrobinson** [comment]
@jagcsun this is not something that is often measured in clinical contexts afaik (see, e.g., https://en.wikipedia.org/wiki/Wallerian_degeneration). Can you provide the context of where this is being used?

**Comment by pnrobinson** [comment]
This is not a phenotypic observation, it is an inference about pathophysiology and not a good HPO term!
